### ISSUE TITLE
Mimic new SQS FiFo feature

### ISSUE THREAD

**Issue by barat** [softwaremill/elasticmq#92]
Hi,

Since Amazon [introduced FiFo in SQS](https://aws.amazon.com/about-aws/whats-new/2016/11/amazon-sqs-introduces-fifo-queues-with-exactly-once-processing-and-lower-prices-for-standard-queues/) I have a question. 
Do You plan to add such feature, or just allow/validate new message parameters for dev purposes?

**Comment by adamw** [comment]
No plans for now, but PRs are always welcome - I just merged & released support for dead letter queues :)

**Comment by osmanpontes** [comment]
Is there a plan for this now?
Having local AWS SQS Fifo Queue would be very nice for test purposes.

**Comment by simong** [comment]
Hi @osmanpontes, FIFO support was introduced in 0.14.0 (#143) and has been slowly improved in later releases. Although the README doesn't call it out explicitly, FIFO should be supported (including message deduping and ordering). Is there anything that's not functioning as you would expect?

**Comment by osmanpontes** [comment]
I was using this docker container: https://hub.docker.com/r/roribio16/alpine-sqs/.
I checked it's using v0.14.6.
In Dockerfile it runs this `curl -sL https://api.github.com/repos/adamw/elasticmq/releases/latest | jq -r .tag_name` which returns v0.14.6.
The queue was missing some sent messages.
But I don't have a reproducible code now.
I changed my application tests to use SQS in AWS infrastructure instead of a mocked version.
On weekend I can try it again and open an appropriated issue.

Thanks for the fast answer @simong !
